meson_options.txt

```diff
@@ -12,7 +12,7 @@ option('HW_ISOLATION_OBJPATH', type: 'string',
       )
 
 option('PHAL_DEVTREE', type: 'string',
-        value : '/var/lib/phosphor-software-manager/pnor/rw/DEVTREE',
+        value : '/var/lib/phosphor-software-manager/hostfw/running/GUARD',
```

Comment:
This configure option is used for the phal device tree not for the guard file.

FYI, this guard manager application rely on the libguard so this change should go into that guard repo.

Reply:
My bad , libgurad already supported this changes.